src/Command/AbstractDependencyGraphCommand.php

```diff
@@ -38,6 +50,7 @@ protected function configure(): void
     {
         $projectRoot = getcwd();
 
+        $this->logger = new ConsoleLogger($output);
```

Comment:
I would put this as a constructor dependency to allow more flexibility in how the output is logged